tools/wpt/browser.py

```diff
@@ -1452,8 +1451,26 @@ class Safari(Browser):
     requirements = "requirements_safari.txt"
 
     def _find_downloads(self):
-        def text_content(e):
-            return etree.tostring(e, encoding="unicode", method="text")
+        def text_content(e, __output=None):
+            # this doesn't use etree.tostring so that we can add spaces for p and br
+            if __output is None:
+                __output = []
+
+            if e.tag == "p":
+                __output.append("\n\n")
+
+            if e.tag == "br":
```

Comment:
Do we actually care about br and p having different amounts of whitespace here? Aren't we really just trying to get one thing per line?

Reply:
In practice, no, but it seemed like little cost to make it perhaps more expected (v. innerText).